docs/source/reconciliation.rst

```diff
@@ -0,0 +1,134 @@
+Reconciliation
+==============
+Reconciliation is the high-level process of modifying an (extended) prefix map with
+domain-specific rules. This is important as it allows for building on existing
+(extended) prefix maps without having to start from scratch. Further, storing the
+rules to transform an existing prefix map allows for high-level discussion about
+the differences and their reasons.
+
+As a specific example, the `Bioregistry <https://bioregistry.io/>`_ uses ``snomedct`` as a preferred prefix for
+the Systematized Nomenclature of Medicine - Clinical Terms (SNOMED-CT). The
+OBO Foundry community prefers to use ``SCTID`` as the preferred prefix for this
+resource. Rather than maintaining a different extended prefix map than the Bioregistry,
+the OBO Foundry community could enumerate its preferred modifications to the base
+(extended) prefix map, then create its prefix map by transforming the Bioregistry's.
+
+Similarly, a consumer of the OBO Foundry prefix map who's implementing a resolver might want to override the URI prefix
+associated with the `Ontology of Vaccine Adverse Events (OVAE) <https://bioregistry.io/registry/ovae>`_
+to point towards the Ontology Lookup Service instead of the default OntoBee.
+
+There are two operations that are useful for transforming an existing (extended) prefix
+map:
+
+1. **Remapping** is when a given CURIE or URI prefix is replaced with another. See :func:`curies.remap`.
```

Comment:
I don't know if maybe you want to change:
> CURIE or URI prefix

to
> CURIE prefix or URI prefix

This would be consistent with your verbiage in the definition for "rewiring" and elsewhere in the document. And also, when I read it, I actually was confused, thinking "CURIE" meant "CURIE" and not "CURIE prefix".

Reply:
There's a 3rd operation that is like the current `remap` that only works on URI prefix -> URI prefix, but I wasn't sure if that was all so useful. I changed the text here to be more clear